exercises/simple-linked-list/Example.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+public class SimpleLinkedList<T>
+{
+    public SimpleLinkedList(T value) :
+        this(value, null)
+    {
+    }
+
+    public SimpleLinkedList(T value, SimpleLinkedList<T> next)
+    {
+        Value = value;
+        Next = next;
+    }
+
+    public T Value { get; }
+
+    public SimpleLinkedList<T> Next { get; }
+
+    public static SimpleLinkedList<T> FromEnumerable(IEnumerable<T> values)
```

Comment:
If this were to match the style of .Net collections, it would make sense to have this as a constructor. That said, I'd like to see the other list methods utilized (`Head`, `Push`, `Pop`, `Peek`) instead of just pulling everything in at once to exercise how a linked list would be used.